app/src/main/java/com/github/se/orator/ui/profile/StatScreen.kt

```diff
@@ -194,9 +194,12 @@ fun LineChart(xValues: List<Int>, yValues: List<Float>, testTag: String, yMin: F
               .padding(start = paddingExtraLarge, top = paddingSmall)
               .testTag(testTag)) {
         // Use the provided yMin and yMax instead of computing from data
-        val minY = yMin
-        val maxY = yMax
-        val yRange = maxY - minY
+        val maxY = yValues.maxOrNull() ?: FULL // full being the const value for 1f
+        val minY = ZERO // zero for the value 0f
+        var yRange = maxY - minY
+        if (yRange < 9f) {
+          yRange = 9f
+        }
```

Comment:
I think it would be a good idea to check with the rest of the team to get their thoughts on what would be more user friendly.

Reply:
Yes you are right.